### ISSUE TITLE
need to add newer HYCOM products

### ISSUE THREAD

**Issue by galuardi** [camrinbraun/HMMoce#17]
*need to add newest HYCOM products to the get.hycom internals*

list is here:

- http://ncss.hycom.org/thredds/catalog.html

PS there may be a gap in coverage December 2018...

**Comment by camrinbraun** [comment]
Still works for me using the dev branch version of ```get.hycom``` as follows:

```
HMMoce:::get.hycom(limits=c(-61,-60,29,30), time=c(as.Date('2019-09-01'), as.Date('2019-09-02')), vars='water_temp', filename = 'try.nc')
```

However, the newer HYCOM products (GLBy.xxx) are 3-hourly which creates a new temporal issue. Added a new temporal "stride" (aka step) to catch this and only get daily HYCOM data in the newer HYCOM experiments: https://github.com/camrinbraun/HMMoce/commit/694f9f5fcde025cd10a356fdf178f4873e81d17b

**Comment by camrinbraun** [comment]
so far this is still up to date. someone shout when HYCOM closes expt 93.0 and moves to the next one
